mod_test.ts

```diff
@@ -949,25 +949,91 @@ const TXT_PLUGIN: esbuildNative.Plugin = {
   },
 };
 
-Deno.test("txt plugin", async (t) => {
+Deno.test("css plugin, entrypoint", async (t) => {
   +await testLoader(t, LOADERS, async (esbuild, loader) => {
     const res = await esbuild.build({
       ...DEFAULT_OPTS,
       plugins: [
-        denoResolverPlugin(),
-        TXT_PLUGIN,
-        denoLoaderPlugin({ loader }),
+        ...denoPlugins({ loader }),
+        CSS_PLUGIN,
       ],
-      entryPoints: ["./testdata/hello.txt"],
+      entryPoints: ["./testdata/basic.css"],
+      bundle: true,
     });
     assertEquals(res.warnings, []);
     assertEquals(res.errors, []);
     assertEquals(res.outputFiles.length, 1);
     const output = res.outputFiles[0];
     assertEquals(output.path, "<stdout>");
     const dataURL = `data:application/javascript;base64,${btoa(output.text)}`;
-    const { default: hello } = await import(dataURL);
-    assertEquals(hello, "Hello World!");
+    const { default: css } = await import(dataURL);
+    assertEquals(css, await Deno.readTextFile("./testdata/basic.css"));
+  });
+});
+
+Deno.test("css plugin, entrypoint with import map", async (t) => {
+  +await testLoader(t, LOADERS, async (esbuild, loader) => {
+    const res = await esbuild.build({
+      ...DEFAULT_OPTS,
+      plugins: [
+        ...denoPlugins({ loader }),
+        CSS_PLUGIN,
+      ],
+      entryPoints: ["$testdata/basic.css"],
```

Comment:
This test case shows deno resolver plugin will resolve paths for files that other plugins will load.